wagtail/images/static_src/wagtailimages/js/focal-point-chooser.js

```diff
@@ -30,6 +30,20 @@ function setupJcrop(image, original, focalPointOriginal, fields) {
       // Set alt="" on the image so its src is not read out loud to screen reader users.
       var $holderImage = $('img', jcropapi.ui.holder);
       $holderImage.attr('alt', '');
+
+      // Create a label tag for the input for accessibility.
+      const label = document.createElement('label');
+
+      label.setAttribute('for', 'jcrop');
```

Comment:
This will not work as expected, `for` must reference an element by its `id` not by its `name`
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for

You will likely need to put an id on the input element that gets generated also, alternatively you can wrap the input with the label element like this
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#simple_label_example

And just put the label content inside a span and put the visuallyhidden class on that span.

`aria-label` on the `input` itself may be suitable here though https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label